extension/linkify-urls-in-code.js

```diff
@@ -14,7 +14,7 @@ window.linkifyURLsInCode = (() => {
 	const linkifyURL = url => `<a href="${url}" ${commonURLAttrs}>${url}</a>`;
 
 	const hasIssue = text => issueRegex.test(text);
-	const hasURL = text => URLRegex.test(text);
+	const hasURL = text => new RegExp(URLRegex).test(text);
```

Comment:
When I don’t have the wrapper, then the global flag doesn’t work properly. As said earlier, I’m not an expert in RegExp, so maybe I’m missing something there.